data/scripts/cable_club.inc

```diff
@@ -1257,7 +1257,14 @@ EventScript_CloseMossdeepGameCornerBarrier::
 	return
 
 CableClub_OnResume:
+.if OW_UNION_CONNECTED_ONLY_CHECK == TRUE
```

Comment:
If `OW_UNION_DISABLE_CHECK` is true, this entire check is pointless because it goes to `CableClub_OnResume_End` either way. I would personally only run this portion of the code if `OW_UNION_DISABLE_CHECK` is false (you could embed it inside a bigger `.if OW_UNION_DISABLE_CHECK == FALSE`

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5050/commits/86593c504a1e65963e8f8e705460460726641a15